esp/src/eclwatch/XrefFoundFilesWidget.js

```diff
@@ -82,15 +161,27 @@ define([
                 var newRows = [];
                 if (lang.exists("DFUXRefFoundFilesQueryResponse.DFUXRefFoundFilesQueryResult.File", response)) {
                     results = response.DFUXRefFoundFilesQueryResponse.DFUXRefFoundFilesQueryResult.File;
+
+                    if (results.length) {
+                        arrayUtil.forEach(results, function (row, idx) {
+                            newRows.push({
+                                Name: row.Partmask,
+                                Modified: row.Modified,
+                                Parts: row.Numparts,
+                                Size: row.Size
+                            });
+                        });
+                    } else if (results.Partmask) {
+                        var temp = {};
+                        lang.mixin(temp, {
+                            Name: results.Partmask,
+                            Modified: results.Modified,
+                            Parts: results.Numparts,
+                            Size: results.Size
+                        })
```

Comment:
If there is a file present (single row) we need to manually insert the row into an array.